### PR TITLE
June and August minutes, and September agenda

### PR DESCRIPTION
**Pull request by weierophinney**
This includes the minutes for the June and August 2021 meetings, and an updated agenda for September 2021.

### CODE REVIEW

**Review by michalbundyra — APPROVED** [review]

**Review by boesing — APPROVED** [review]
LGTM
I think there is nothing essential missing tho.
Not sure about the `GITHUB_TOKEN` discussion as I did not really follow.

File: meetings/minutes/2021-06-07-TSC-Minutes.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# 2021-06-07 TSC Meeting
+
+- When: 2021-06-07 at 19:00 UTC, until 20:48 UTC
+- Attending:
+  - Abdul Malik Ikhsan
+  - Aleksei Khudiakov
+  - Frank Brüchner
```
Comment (froschdesign) [inline meetings/minutes/2021-06-07-TSC-Minutes.md:7]:
```suggestion
  - Frank Brückner
```

File: meetings/minutes/2021-06-07-TSC-Minutes.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - Geert Eltink
+  - Luís Cobucci
+  - Marco Pivetta
+  - Matthew Setter
+  - Matthew Weier O'Phinney
+  - Max Bösing
```
Comment (froschdesign) [inline meetings/minutes/2021-06-07-TSC-Minutes.md:13]:
```suggestion
  - Maximilian Bösing
```

Comment (boesing) [reply meetings/minutes/2021-06-07-TSC-Minutes.md:13]: thx, even tho Max would be fine as well 😄👍🏻

File: meetings/minutes/2021-06-07-TSC-Minutes.md (hunk 1 above)
Comment (weierophinney) [reply meetings/minutes/2021-06-07-TSC-Minutes.md:7]: I'm so sorry - I'm constantly getting this one wrong. :-/

Comment (froschdesign) [reply meetings/minutes/2021-06-07-TSC-Minutes.md]: No problem at all!

**Pull request merged**